scalafix-core/shared/src/main/scala/scalafix/internal/config/DisableUnlessConfig.scala

```diff
@@ -0,0 +1,62 @@
+package scalafix.internal.config
+
+import metaconfig.{Conf, ConfDecoder, ConfError, Configured}
+import org.langmeta.Symbol
+
+import scalafix.internal.config.MetaconfigPendingUpstream.XtensionConfScalafix
+import scalafix.internal.util._
+
+case class UnlessConfig(
+    block: Symbol.Global,
+    symbol: Symbol.Global,
```

Comment:
Might want to make this a `List[Symbol.Global]` down the road in order to reuse lists of bad symbols